### ISSUE TITLE
Doesn’t work with HA 2024.1.0

### ISSUE THREAD

**Issue by the-ress** [h4de5/home-assistant-toshiba_ac#166]
I see the fix is already merged, but not released: #163

```
Logger: homeassistant.loader
Source: loader.py:842
First occurred: 9:30:37 PM (1 occurrences)
Last logged: 9:30:37 PM

Unexpected exception importing platform custom_components.toshiba_ac.select
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 842, in get_platform
    cache[full_name] = self._import_platform(platform_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 859, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/toshiba_ac/select.py", line 106, in <module>
    ToshibaAcEnumSelectDescription(
TypeError: ToshibaAcEnumSelectDescription.__init__() missing 1 required keyword-only argument: 'icon_mapping'
```

**Comment by h4de5** [comment]
should be fixed in 2024.1 release

**Comment by the-ress** [comment]
thanks, I can confirm it’s fixed
